src/util/tracking.js

```diff
@@ -86,11 +86,23 @@ async function logToFirebase(event: TrackingEvent, values: TrackingValues) {
   }
 }
 
+export async function multicastUtilServers(path: string, options?: Object): Promise<any> {
+  const serverUrls = ['https://util2.edge.app/', 'https://util1.edge.app/']
+  let reply
+  for (const url of serverUrls) {
+    reply = fetch(`${url}${path}`, options)
+    if (reply.ok) {
+      return reply
```

Comment:
This will never get hit since reply is a promise and you do not have an await Before the call to fetch